src/Dropdown.elm

```diff
@@ -414,7 +432,10 @@ triggerView config state =
                         )
                         { onChange = config.dropdownMsg << OnFilterTyped
                         , text = state.filterText
-                        , placeholder = Just <| Input.placeholder [] (text config.filterPlaceholder)
+                        , placeholder =
```

Comment:
I think this should be config.filterPlaceholder |> Maybe.map (text |> Input.placeholder []) |> Maybe.withDefault none, becvause in the current one you could end with
Just (Input.placeholder [] none)

Reply:
I don't know if this last one is correct

Reply:
So you mean i should have instead of Just (Input.placeholder [] none) I should have Nothing, right?..

Reply:
sorry, wasn't clear. Yes, that precisely. I wonder if it is better to have

Just (input.placeholder [] none) 

or just simply to put a 

Nothing in the placeholder

as both will render nothing to the screen

Reply:
I think this does the work. Right? If filterPlaceholder is Nothing then Nothing for the Input